### PR TITLE
Configuration commands table converted to list view.

### PR DESCRIPTION
**Pull request by Spacewalker2**

### CODE REVIEW

File: doc/configcommands-linked.asciidoc [hunk 1]
```diff
@@ -0,0 +1,1624 @@
...
+<<urls-source>> +
+<<urlview-title-format>> +
+<<use-proxy>> +
+<<user-agent>> +
+
+
+
+'''
+****
+[[always-display-description]]*Syntax:* always-display-description [yes/no] +
```
Comment (Minoru) [inline doc/configcommands-linked.asciidoc:142]: I imagined that each item will link to itself, something like:

```
[[always-display-description]]*Syntax*: <<always-display-description>> [yes/no]
```

This way I can find the setting I want, click its link, and copy-paste that into a chat or whatever. With current layout, I have to go hunt for the link above the list, which is inconvenient.
```

**Comment by Minoru** [comment]
(Linking this to issue #441).

**Comment by Minoru** [comment]
Also, don't throw out the scripts you used to prepare this, because #936 already adds a new setting, and I'm not sure how many more we'll add before this PR is merged. Though perhaps a couple settings can be added by hand as well.

**Comment by Spacewalker2** [comment]
The script is very simple and uses the dsv file too so I think it could be integrated in the build process in some way. But before that it needs one or two minor modifications.

**Pull request closed without merging**